frontend/assets/style/_reset.scss

```diff
@@ -34,7 +34,7 @@ h3,
 h4,
 h5,
 h6 {
-  font-weight: bold;
+  font-weight: normal;
```

Comment:
Wow look like a big change for that pull request.. but I guess you've tested everywhere where there is a title that it look like the design, isn't it?

Reply:
Yes, I did exactly that on the last PR where we changed `is-X` to `is-title-X`, so now, *every* title must have the `is-title-x` class, so there's no need to add `font-weight: bold` to heading elements. When we need to use a heading element without looking like a title, we don't need to override its style anymore.